airbyte-connector-builder-server/src/main/openapi/openapi.yaml

```diff
@@ -144,6 +144,9 @@ components:
           type: object
           description: The AirbyteStateMessage object to use as the starting state for this read
           # $ref: "#/components/schemas/AirbyteProtocol/definitions/AirbyteStateMessage"
+        record_limit:
```

Comment:
@lmossman do we need to regenerate any typescript code? (if yes we should make it more obvious/automatic for anyone touching this code)

Reply:
The generated typescript code is not committed to github, and since this is just adding a new optional parameter it won't break any of our code that consumes that generated typescript code. So this shouldn't require any frontend changes